cartridges/int_algolia/cartridge/scripts/algolia/steps/algoliaContentIndex.js

```diff
@@ -8,16 +8,15 @@ var logger;
 var paramAttributeList, paramFailureThresholdPercentage, includedContent;
 
 // Algolia requires
-var algoliaData, AlgoliaLocalizedContent, jobHelper, reindexHelper, algoliaIndexingAPI, contentFilter, AlgoliaJobReport, algoliaSplitter, algoliaContentConfig;
+var algoliaData, AlgoliaLocalizedContent, jobHelper, reindexHelper, algoliaIndexingAPI, contentFilter, AlgoliaJobReport, algoliaSplitter, algoliaContentConfig, ContentUtil;
 var indexingOperation;
 
 // logging-related variables
 var jobReport;
 
 var contents = [], siteLocales, nonLocalizedAttributes = [], attributesToSend, count;
 var lastIndexingTasks = {};
-
-
+var enableContentFunctionHandler = true;
```

Comment:
Let's remove the parameter entirely since the performances are not too impacted in the end, wdyt?

Reply:
yes it is fine, will remove it.